README.md

```diff
@@ -0,0 +1,15 @@
+# gardener-extension-csi-driver-lvm
+
+Provides a Gardener extension for managing [csi-driver-lvm](https://github.com/metal-stack/csi-driver-lvm) for a shoot cluster.
+
+The extension checks for the old [csi-lvm](https://github.com/metal-stack/csi-lvm/tree/master) and removes it.
```

Comment:
Not sure if the extension can simply remove it as it is applied by the gardener-resource-manager consistently. It will be re-deployed after some seconds. I would assume a user first needs to disable the deployment through a gardener-extension-provider-metal, which could be put behind a feature gate, and then this extension can be added? In case the old storage class is still present, the reconcile should probably stop? Or do you have a different plan?

Reply:
You are right. First thought, that resource is deployt manually and not by extension. Created a document describing problems and solutions with the migration. Your workflow got added to the document: See [docs/migration.md](https://github.com/metal-stack/gardener-extenstion-csi-driver-lvm/pull/1/files#diff-758ba107a728fccfdf0d098b2c06ded59127dd53ea287d630f72fb51040bc72d)